debian/source/format

```diff
@@ -0,0 +1 @@
+3.0 (native)
```

Comment:
Prefer `3.0 (quilt)`, I think.

Reply:
I'm not a big fan of `3.0 (quilt)`, since it would require a separate `debian-master` branch and needs pristine tarballs to build. We are the only consumers of this package, so it just makes sense to keep the packaging in sync with the main history using `3.0 (native)`.

Reply:
It doesn't require a separate branch.